byron/chain/executable-spec/test/Cardano/AbstractSize/Properties.hs

```diff
@@ -0,0 +1,163 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Cardano.AbstractSize.Properties
+    (testAbstractSize)
+  where
+
+import           Data.AbstractSize
+import           Data.Map.Strict (Map)
+import qualified Data.Map.Strict as Map
+import           Data.Maybe (maybeToList)
+import           Data.Sequence ((<|), (><))
+import qualified Data.Sequence as Seq
+import qualified Data.Set as Set
+import           Data.Typeable (TypeRep, Typeable, typeOf)
+import           Data.Word (Word64)
+import           Numeric.Natural (Natural)
+
+import           Hedgehog (MonadTest, Property, forAll, property, withTests, (===))
+import           Test.Tasty.Hedgehog
+
+import           Control.State.Transition.Generator (trace)
+import           Control.State.Transition.Trace (TraceOrder (OldestFirst), traceSignals)
+import           Ledger.Core hiding ((<|))
+import           Ledger.Delegation (DCert)
+import           Ledger.Update (ProtVer (..), STag, UProp (..), Vote)
+import           Ledger.UTxO
+
+import           Cardano.Spec.Chain.STS.Block (Block (..), BlockBody (..), BlockHeader (..))
+import           Cardano.Spec.Chain.STS.Rule.Chain (CHAIN)
+
+import           Test.Tasty (TestTree, testGroup)
+import           Test.Tasty.HUnit (Assertion, testCase, (@?=))
+
+--------------------------------------------------------------------------------
+-- Example typeReps for Block/Header/Body
+--------------------------------------------------------------------------------
+
+aTx :: Tx
+aTx = undefined
+
+aTxId :: TxId
+aTxId = TxId (hash aTx)
+
+aHeader :: BlockHeader
+aHeader = MkBlockHeader {
+          _bhPrevHash = (undefined :: Hash)
```

Comment:
The brackets aren't needed.

Reply:
There are a couple of other unnecessary brackets in the file as well. See if you'd like to run `hlint` on the PR files :)

Reply:
my hlint was not working for a few days, and it shows ;)